tools/autograd/templates/Functions.cpp

```diff
@@ -751,6 +751,22 @@ Tensor cholesky_backward(Tensor grad, bool upper, Tensor L) {
   return grad_input.add(grad_input.transpose(-1, -2)).mul_(0.5);  // Symmetrizing the gradient
 }
 
+Tensor cholesky_inverse_backward(Tensor grad, Tensor L, bool upper, Tensor inverse) {
+  Tensor grad_L;
+  if (grad.defined()) {
+    Tensor common_term = grad + grad.transpose(-2, -1);
+    common_term = at::matmul(inverse, at::matmul(common_term, inverse));
+    if (upper) {
+      grad_L = -at::matmul(inverse, common_term);
+    } else {
+      grad_L = -at::matmul(common_term, inverse);
```

Comment:
Maybe I am wrong because the tests pass, but from the reference, I think this is `-at::matmul(common_term, L)`

Reply:
Oh yes, I botched that in a renaming update. Thanks for pointing it out!